package.json

```diff
@@ -11,11 +11,17 @@
     "test": "mocha tests/ --reporter @digitalbazaar/mocha-w3c-interop-reporter --reporter-options abstract=\"$PWD/abstract.hbs\",reportDir=\"$PWD/reports\",respec=\"$PWD/respecConfig.json\",suiteLog='./suite.log',templateData=\"$PWD/reports/index.json\",title=\"Data Integrity ecdsa 2019 Interoperability Report 1.0\" --timeout 15000 --preserve-symlinks",
     "lint": "eslint ."
   },
+  "license": "BSD-3-Clause",
   "author": {
-    "name": "W3C Verifiable Credentials Working Group",
+    "name": "W3C Verifiable Credentials Working Group Contributors",
```

Comment:
```suggestion
    "name": "W3C, Inc.",
```

Reply:
I think the actual entity name is "World Wide Web Consortium" per the https://w3.org/ copyright. I'll update it regardless.

Reply:
And actually... This is an authorship claim, not a copyright claim per se and I think it makes the rest of the author metadata tangled. I'll update the copyright claim in the LICENSE file, but I think this one should stay as it is here.